src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -221,7 +221,7 @@
     dnf:
       state: present
       name: sssd-kcm
-  when: "'base_ipa' in group_names or 'ipa' in group_names"
+  when: "'base_ipa' in group_names or 'base_ipa2' in group_names or 'ipa' in group_names"
```

Comment:
Is there a reason to use base_ipa2? Wouldn't it be tha same as base_ipa?

Reply:
Fixed to use `base_ipa` only, this was an oversight on my part.

Reply:
Does not look fixed yet.